provision-enterprise-access.sh

```diff
@@ -0,0 +1,28 @@
+set -euf -o pipefail
+
+. scripts/colors.sh
+
+name="enterprise-access"
+port="18270"
+
+docker-compose up -d $name
+
+# Run migrations
```

Comment:
Are we going to use the following convention here
https://github.com/edx/devstack/blob/master/provision-notes.sh#L15
If yes, then we need to remove this line
https://github.com/edx/public-dockerfiles/blob/main/dockerfiles/enterprise-access.Dockerfile#L121